stations.csv

```diff
@@ -17422,7 +17422,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 19427;Bari Santo Spirito;bari-santo-spirito;8311116;83111161;41.159772;16.746409;22157;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8301025;f;;f;;f;;f;;f;;f;8301025;f;;f;8311116;t;S11116;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;バーリ;바리;;;Бари;;;巴里;urn:trainline:public:nloc:csv19427;
 19428;Bari Zona Industriale;bari-zona-industriale;;;41.105567;16.829262;22157;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8302467;f;;f;;f;;f;;f;;f;8302467;f;;f;8311156;t;S11120;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;バーリ;바리;;;Бари;;;巴里;urn:trainline:public:nloc:csv19428;
 19429;Baricetta;baricetta;8305419;83054197;45.060847;11.995547;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8301174;f;;f;;f;;f;;f;;f;;f;;f;8305419;t;S05419;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv19429;
-19430;Barletta;barletta;8311108;83111088;41.315402;16.278628;;IT;Europe/Rome;f;f;f;t;f;f;ITATI;;f;;f;8300273;f;;f;ITBAL;t;8648;t;;f;;f;8300273;f;;f;8311108;t;S11108;;;BLT;t;;f;;f;;f;ITATI;f;;f;f;;;;;;;;;;;;バルレッタ;;;;Барлетта;;;;urn:trainline:public:nloc:csv19430;
+19430;Barletta;barletta;8311108;83111088;41.315402;16.278628;;IT;Europe/Rome;t;f;f;t;f;f;ITATI;;f;;f;8300273;f;;f;ITBAL;t;8648;t;;f;;f;8300273;f;;f;8311108;t;S11108;;;BLT;t;;f;;f;;f;ITATI;f;;f;f;;;;;;;;;;;;バルレッタ;;;;Барлетта;;;;urn:trainline:public:nloc:csv19430;
```

Comment:
:thinking: I'm not convinced it is an appropriate change:
- The coordinates of this station seem to point to the railway station. If we would want to add a city, it should be a different station.

- `Barletta Centrale FNB` seem to be pointing at a coach station adjacent to the railway station, so it is okay-ish to have it as a child of Barletta.

Known all that, I don't think it is appropriate to have `Barletta Scalo` as a child station of Barletta

Reply:
:+1: sounds good. We might add a separate station for the city later on, if needed.